app.js

```diff
@@ -173,36 +201,36 @@ function createRow(entry, tr, games, teams) {
   possiblePointsTd.innerHTML = possible_points;
   possiblePointsTd.dataset.heading = "max points";
 
-  games.forEach((game) => {
-    let gameTd = document.createElement("td");
-    let inner = document.createElement("div");
+  for (const game of games) {
+    const gameTd = document.createElement("td");
+    const inner = document.createElement("div");
     gameTd.appendChild(inner);
     inner.classList.add("inner");
-    let selectedPick = document.createElement("img");
+    const selectedPick = document.createElement("img");
     inner.appendChild(selectedPick);
 
     gameTd.classList.add("narrow");
     gameTd.classList.add("logo");
 
     const homeTeam = teams.find((team) => team.team_id == game.team_1_id);
     const awayTeam = teams.find((team) => team.team_id == game.team_2_id);
-    gameTd.dataset.heading = `${homeTeam?.abbreviation || "?"} - ${
-      awayTeam?.abbreviation || "?"
-    }`;
+    gameTd.dataset.heading = `${homeTeam?.abbreviation || "?"} - ${awayTeam?.abbreviation || "?"
+      }`;
 
     const gameId = game.id;
     const pickKey = `match_${gameId}_pick`;
     const userPick = entry[pickKey];
 
     // If user's pick is not in the running anymore
-    if (userPick != game.team_1_id && userPick != game.team_2_id) {
-      selectedPick.src = `dash.svg`;
+    if (userPick !== game.team_1_id && userPick !== game.team_2_id) {
```

Comment:
The APIs are quite inconsistent as how they store different values and thus the `==` was intentional here as I was lazy to convert everything to a consistent one.

This change currently breaks the app as the `userPick` and `game.[team_id]` are not of the same type.

Reply:
Yeah just noticed this myself also, will revert.